packages/orderbook/package.json

```diff
@@ -12,6 +12,9 @@
         "type": "git",
         "url": "https://github.com/0xProject/0x-monorepo.git"
     },
+    "publishConfig": {
+        "access": "public"
```

Comment:
I'm not familiar with this package. Was there a reason it was private before and now should be public? Or was it just an unintentional omission?

Reply:
it was accidentally omitted and killed the publish :(.